pkgs/applications/misc/font-manager/default.nix

```diff
@@ -42,6 +42,10 @@ stdenv.mkDerivation rec {
     "-Ddisable_pycompile=true"
   ];
 
+  patches = [
+    ./fix-build-vala-0.46.patch
```

Comment:
```suggestion
    # fix build with Vala 0.46
    (fetchpatch {
      url = "https://github.com/FontManager/font-manager/commit/c73b40de11f376f4515a0edfe97fb3721a264b35.patch";
      sha256 = "0lacwsifgvda2r3z6j2a0svdqr6mgav7zkvih35xa8155y8wfpnw";
      excludes = [ "fedora/font-manager.spec" ];
    })
```

Reply:
Yep, forgot about that useful functionality :+1: